pkg/services/container/morph/executor.go

```diff
@@ -80,6 +82,11 @@ func (s *morphExecutor) Put(_ context.Context, tokV2 *sessionV2.Token, body *con
 		if err != nil {
 			return nil, fmt.Errorf("invalid session token: %w", err)
 		}
+
+		err = s.validateToken(tokV2, nil, sessionV2.ContainerVerbPut)
+		if err != nil {
+			return nil, fmt.Errorf("token validation: %w", err)
```

Comment:
if client-facing, then
```suggestion
			return nil, fmt.Errorf("session token validation: %w", err)
```

Reply:
changed but the user will not see it because of how we unwrap the errors (status gets the lowest one only)